apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -137,4 +149,31 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
   # `exitable` will be `false` ALONG WITH the whole tuple holding zeroees, if the exit was processed successfully
   # **NOTE** one can only rely on the zero-nonzero of this data, since for processed exits this data will be all zeros
   defp parse_contract_exit_status({exitable, _, _, _, _, _}), do: exitable
+
+  @doc """
+  Based on the block number determines whether UTXO was created by a deposit.
+  """
+  defguard is_deposit(blknum, child_block_interval) when rem(blknum, child_block_interval) != 0
```

Comment:
I think this can be a private function since we don't use it in pattern matching and it's only used locally in this module.

Reply:
b186e8a